library/model_support.c

```diff
@@ -349,6 +414,53 @@ static int parse_array(void **arr, const char *json, jsmntok_t *tok, type_meta *
     return processed;
 }
 
+static int parse_map(void *mapp, const char *json, jsmntok_t *tok, type_meta *el_meta) {
+    if (tok->type != JSMN_OBJECT) {
+        ZITI_LOG(ERROR, "unexspected JSON token near '%.*s', expecting object", 20, json + tok->start);
```

Comment:
i didn't expect to see `unexspected` here

Reply:
it means not up to spec, duh!